lisa/sut_orchestrator/azure/tools.py

```diff
@@ -62,22 +62,27 @@ def get_generation(self) -> str:
 
 class LisDriver(Tool):
     """
-    This is a virtual tool to detect/install LIS (Linus Integration Services) drivers.
+    This is a virtual tool to detect/install LIS (Linux Integration Services) drivers.
     More info  - https://www.microsoft.com/en-us/download/details.aspx?id=55106
     """
 
     __version_pattern = re.compile(r"^version:[ \t]*([^ \n]*)")
 
+    @property
+    def command(self) -> str:
+        return ""
+
     @property
     def dependencies(self) -> List[Type[Tool]]:
         return [Wget, Modinfo]
 
     @property
     def can_install(self) -> bool:
-        if (
-            isinstance(self.node.os, Redhat)
-            and float(self.node.os.os_version.release) < 7.8
-        ):
+        if len(self.node.os.os_version.release.split(".")) > 2:
```

Comment:
Add comments here to explain the logic. BTW, I think we can split versions to an int/str array, and use a method to compare it. For example, 8.3 can be [8, 3], and 8.3.20210609 can be [8, 3, 20210609]. They are easy to compare within same format. Anyway, you can add a task to me to support it.

Reply:
Since the old logic was only considering version if isinstance(self.node.os, Redhat) , I think we don't need to worry about the version with format *.*.*

Reply:
> Add comments here to explain the logic. BTW, I think we can split versions to an int/str array, and use a method to compare it. For example, 8.3 can be [8, 3], and 8.3.20210609 can be [8, 3, 20210609]. They are easy to compare within same format. Anyway, you can add a task to me to support it.

Despite that we don't need to worry about the os version format issue here, I agree we can make version like you suggested. I will push out a PR for this today.

Reply:
It can be a Version class, it supports `<`, `==`, `>`, `raw` methods.

Reply:
Let's not re-invent the wheel: https://pypi.org/project/semver/